automation/workflow/wftest/wftest.go

```diff
@@ -0,0 +1,263 @@
+package wftest
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/jinzhu/copier"
+	"github.com/qri-io/dataset"
+	"github.com/qri-io/qri/automation/workflow"
+	"github.com/qri-io/qri/base/dsfs/dstest"
+	"github.com/qri-io/qri/lib"
+	"github.com/qri-io/qri/profile"
+)
+
+const (
+	// InputWorkflowFilename is the filename to use for an input workflow
+	InputWorkflowFilename = "input.workflow.json"
+	// ExpectWorkflowFilename is the filename to use to compare expected outputs
+	ExpectWorkflowFilename = "expect.workflow.json"
+)
+
+var (
+	defaultTestCasesDir = ""
+	testCaseCache       = []*TestCase{}
+)
+
+// TestCase is a workflow test case, usually built from a
+// directory of files for use in tests
+// Each workflow for each test case must have an associated
+// dataset
+type TestCase struct {
+	// Path to the directory on the local filesystem this test case is loaded from
+	Path string
+	// Name is the casename, should match the directory name
+	Name string
+	// Input is intended file for test input
+	// loads from input.workflow.json
+	Input *workflow.Workflow
+	// Expect should match the test output
+	// loads from expect.workflow.json
+	Expect  *workflow.Workflow
+	Dataset *dstest.TestCase
+}
+
+// LoadDefaultTestCases loads test cases from this package
+// the OwnerID is the first key in the `auth/key/test/keys.go`, which is the
+// primary key used in the default test configs
+func LoadDefaultTestCases() ([]*TestCase, error) {
```

Comment:
The default way to get workflow test cases!

The test cases currently in the `wftest/testdata` directory were primarily created for testing within the `wftest` pacakge and as a model for other packages that want to load from a directory.